user-docs/latest/content-manager/introduction-to-content-manager.md

```diff
@@ -0,0 +1,91 @@
+
+To set a new filter:
+
+1. Click on the **Filters** button.
+2. Click on the 1st listbox to choose the field on which the condition will be applied.
+3. Click on the 2nd listbox to choose the type of condition to apply.
+4. Enter the value of the condition in the 3rd box.
+5. (optional) Click on the add button ![icon add new](../assets/content-manager/icon_add2.png) to add another condition-based filter.
+6. Click on the **Apply** button.
+
+::: tip NOTE
+When active, filters are displayed next to the **Filters** button. They can be removed by clicking on the delete icon ![icon delete](../assets/content-manager/icon_delete2.png).
+:::
+
+### Creating a new entry
+
+On the top right left side of the list view interface, an **Add New [collection type name]** button is displayed. It allows to create a new entry for your collection type.
+
+Clicking on the new entry button will redirect you to the edit view, where you will be able to write the content of the new entry (see [Writing content](writing-content.md)).
+
+::: tip NOTE
+New entries are only considered created once some of their content has been written and saved once. Only then will the new entry be listed in the list view.
+:::
+
+
+### Configuring the table fields
+
+Right above the list view table, on the right side of the interface, a settings button <Fa-Cog /> is displayed. It allows to access the configurations that can be set for the list view of your collection type (see [Configuring view of content type](../content-manager/configuring-view-of-content-type.md)), and to choose which fields to display in the table.
+
+::: tip NOTE
+Configuring the displayed field of the table in the way detailed below is only temporary: the configurations will be resetted as soon as the page is refreshed or when navigating the admin panel outside the Content Manager. For permanent configurations, please refer to [Configuring view of content type](../content-manager/configuring-view-of-content-type.md).
+:::
+
+![Displayed fields in the settings of a list view in the Content Manager](../assets/content-manager/content-manager_displayed-fields.png)
+
+To temporarily configure the fields displayed in the table:
+
+1. Click on the settings button <Fa-Cog />.
+2. In the Displayed Fields section, tick the boxes associated with the field you want to be displayed in the table.
+3. Untick the boxes associated with the fields you do not want to be displayed in the table.
+
+::: tip NOTE
+Relational fields can also be displayed in the list view. There are however some specificities to keep in mind:
+
+- Only one field can be displayed per relational field.
+- Only first-level fields can be displayed (i.e. fields from the relation of a relation can't be displayed).
+- If the displayed field contains more than one value, not all its values will be displayed, but a counter indicating the number of values. You can hover this counter to see a tooltip indicating the first 10 values of the relational field.
```

Comment:
I don't know if it's needed to specify that the 10 first value will be cropped if they are too big. Meaning if 1 value is longer than 1 line, than it'll be truncated.

Reply:
I thought about that, but I feel like without enough context it would be too difficult to understand